src/app/view/templates/beol/beol.component.ts

```diff
@@ -116,23 +116,23 @@ export class BeolComponent implements OnInit {
 
         const Christian_Wolf = new Correspondent("Christian Wolff", "(DE-588)118634771");
 
-        const Johann_Frick = new Correspondent("Johann Frick", "(DE-588)123393094");
+        const Johann_Frick = new Correspondent("Johannes Frick", "(DE-588)123393094");
 
         const Henri_Basnage_de_Beauval = new Correspondent("Henri Basnage de Beauval", "(DE-588)12053827X");
 
         const Christoph_Battier = new Correspondent("Christoph Battier", "(DE-588)141546573");
 
         const Georg_Bernhard_Bilfinger = new Correspondent("Georg Bernhard Bilfinger", "(DE-588)118663208");
 
-        const Johann_Wendel_Bilfinger = new Correspondent("Johann Wendel Bilfinger", "(DE-588)104171057");
+        const Johann_Wendel_Bilfinger = new Correspondent("Johann Wendelin Bilfinger", "(DE-588)104171057");
 
         const Pierre_Bouguer = new Correspondent("Pierre Bouguer", "(DE-588)117622974");
 
-        const Marc_Michel_Bousquet = new Correspondent("Marc Michel Bousquet", "(DE-588)136952968");
+        const Marc_Michel_Bousquet = new Correspondent("Marc-Michel Bousquet", "(DE-588)136952968");
 
         const William_Burnet = new Correspondent("William Burnet", "(DE-588)104185643");
 
-        const Johann_Buxtorf = new Correspondent("Johann Buxtorf", "(DE-588)12068943X");
```

Comment:
The name was correct as it was before  (DE-588)12068943X corresponds to Johann Buxtorf both in name index and in DNB
https://portal.dnb.de/opac.htm?method=showFullRecord&currentResultId=12068943X%26any&currentPosition=3

Reply:
done in https://github.com/dhlab-basel/Salsah/pull/206/commits/a1e7e6768121de38743b51670be2e1168d3b6395